README.md

```diff
@@ -24,25 +24,34 @@ one that is at least version `0.55.0`).
 
 Clone this repository:
 ```
-$ git clone https://github.com/rizinorg/rizin
+git clone https://github.com/rizinorg/rizin
 ```
 
 Then compile and install with:
 ```
-$ meson setup build
-$ meson compile -C build
-$ sudo meson install -C build
+sudo apt install meson -y 
+```
+```
+meson setup build
+```
+```
+meson compile -C build
+```
+```
+sudo meson install -C build
```

Comment:
there is no real reason to have this splitted.
also add `bash` on the same line to have colored syntax.

Reply:
> there is no real reason to have this splitted.

😑 yeah why not!